### ISSUE TITLE
Missing curl in sidecar image

### ISSUE THREAD

**Issue by RasmusGodske** [SchwarzIT/node-red-chart#149]
### What happened?

flow_refresh.sh uses curl, however curl does not exist in sidecar image.

### How can we reproduce this?

Enable sidecar in the values.yaml and provide nodered username and password.

### Helm Chart version

```bash
0.20.0
```

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dirien** [comment]
Hi @RasmusGodske,

thanks for raising the issue! Let me check if it makes sense to create a custom image wich contains curl and provide it as part of the helm this helm chart!

**Comment by RasmusGodske** [comment]
No worries, we did find a way around it by extending the image as seen here [RasmusGodske/k8s-sidecar-curl-yq](https://github.com/RasmusGodske/k8s-sidecar-curl-yq), and use that image instead for the sidecar.

However we did end up taking a completely different route, since the sidecar would fail to write to the flows.json file, after you have updated the flows through the node-red webui. We are not entirely sure why this happend, but after the node-red container updated the flows.json file the file would become read-only within the sidecar container.
